cmd/node/main.go

```diff
@@ -181,6 +183,25 @@ func main() {
 	}()
 	go Discovering(csiNodeService, logger)
 
+	// check here for volume manager readiness
+	// input parameters are ignored by Check() function - pass empty context and health check request
+	ctx := context.Background()
+	req := &grpc_health_v1.HealthCheckRequest{}
+	for {
```

Comment:
+1

Reply:
It will stuck in Unready state. We can't proceed without it. If we limit number of retries - then need to exit, but node pod restart won't solve the problem with drive manager.

Reply:
Number of retries added.